planet_crs_registry/core/business/crs_response.py

```diff
@@ -49,54 +49,39 @@ class GmlResponse(Response):
 
     media_type = "application/xml"
 
-    def render(self, content: str) -> bytes:
-        """Renders the GML response.
-        The GML response is renders in two different ways:
-        - in prod : use gdalsrsinfo because the web service is installed
-        in the same image as gdal
-        - in dev : use gdal image
+    @staticmethod
+    def render(content: str) -> bytes:
+        """Render the GML response from stored data.
 
         Args:
-            content (str): IAU identifier
+            content (str): IAU identifier (separated by underscore)
 
         Returns:
             bytes: the response in GML
         """
-        data: bytes
-        if IS_PROD:
-            data = subprocess.check_output(
-                [
-                    "gdalsrsinfo",
-                    content,
-                    "-o",
-                    "xml",
-                ]
-            )
-        else:
-            data = subprocess.check_output(
-                [
-                    "docker",
-                    "run",
-                    "--rm",
-                    "osgeo/gdal",
-                    "gdalsrsinfo",
-                    content,
-                    "-o",
-                    "xml",
-                ]
+        gml_path = f"/planet_crs_registry/data/gml/{content}.xml"
```

Comment:
Mmh... ça, c'est potentiellement un chemin absolu, ça me paraît dangereux.

Je pense qu'il y a qqch à creuser sur la gestion des resources/data.

Reply:
Solved in pull request #3